site/app/templates/polls/PollForm.twig

```diff
@@ -87,7 +87,15 @@
             <input type="checkbox" id="keep-image" name="keep_image" checked />
         {% endif %}
         <br/>
-
+        
+        <br/>
+        <label for="poll-hours" class="option-title">
+            Expected Duration of Poll:
+        </label>
+        <input type="number" name="poll-hours" id="poll-hours" class="poll-timer" style="width: 70px" min=0 placeholder="Hour" value="{{ poll is not null ? poll.getDuration().h }}" />
+        <input type="number" name="poll-minutes" id="poll-minutes" class="poll-timer" style="width: 70px" min=0 placeholder="Min" value="{{ poll is not null ? poll.getDuration().i }}" />
+        <input type="number" name="poll-seconds" id="poll-seconds" class="poll-timer" style="width: 70px" min=0 placeholder="Sec" value="{{ poll is not null ? poll.getDuration().s }}" />
```

Comment:
There should also be an indication if a poll timer is set to infinity rather than just empty fields - maybe a button to toggle on/off the feature?

Reply:
I don't quite follow, the instructors can see which boxes are hours minutes and seconds because of the placeholder. If the instructor doesn't do anything the poll-timer doesn't do anything, the functionality of the poll remains unchanged. Therefore, I don't think we need to set an on/off feature it is just an optional feature they can use if they want.